pallets/msa/src/lib.rs

```diff
@@ -151,10 +151,11 @@ pub mod pallet {
 
 	/// Storage type for key to MSA information
 	/// - Key: AccountId
-	/// - Value: [`KeyInfo`](common_primitives::msa::KeyInfo)
+	/// - Value: [`MessageSourceId`]()
 	#[pallet::storage]
 	#[pallet::getter(fn get_key_info)]
-	pub type KeyInfoOf<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, KeyInfo, OptionQuery>;
+	pub type KeyInfoOf<T: Config> =
```

Comment:
Should this still be named `KeyInfoOf`?

Reply:
Good question. We could go with something like `MessageSourceIdOf` if it isn't too much of a mouthful.